### DISCUSSION TITLE
Twitter player cards are no longer working

### DISCUSSION THREAD

**Discussion by futahei** [google/model-viewer#4071] [General]
This is not a model-viewer issue, but I am letting you know here that modelviewer.dev offers a Twitter Card generator.

For some reason, Twitter Player Card is not working now. Of course, the preview in model-viewer.dev is not working anymore.

This preview is very interesting and we are looking forward to having it available again.

Forgive the strange English in the translation tool &-(

**Comment by futahei** [comment]
Here is a related post.
[Player cards no longer working](https://twittercommunity.com/t/player-cards-no-longer-working/184308?u=kohei_fttk)

**Reply by elalish** [reply]
Indeed, and since that one's deleted, here's another: https://twittercommunity.com/t/player-cards-no-longer-working/184310

Sounds like they're aware of the issue, so hopefully fixed soon. At least it's not just us.

**Reply by elalish** [reply]
Looks like it's working again.

**Comment by rawwerks** [comment]
can someone link to a tweet that uses the twitter player card for modelviewer? i'm not getting the modelviewer part working, only the poster image.

**Reply by elalish** [reply]
Ours still seems to be working: https://twitter.com/modelviewer/status/1450498949475799040

**Reply by leocodeio** [reply]
can you share the code and the link you have used to go to x so that it have rendered the thing which looks like you post?

**Reply by elalish** [reply]
https://modelviewer.dev/docs/faq.html#entrydocs-general-questions-twitter
